release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4372,3 +4372,34 @@ $ oc adm release info 4.13.50 --pullspecs
 [id="ocp-4-13-50-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.51
+[id="ocp-4-13-51"]
+=== RHSA-2024:6811 - {product-title} 4.13.51 bug fix and security updates
+
+Issued: 25 September 2024
+
+{product-title} release 4.13.51, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6811[RHSA-2024:6811] advisory. TThe RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:6813[RHBA-2024:6813] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.51 --pullspecs
+----
+
+[id="ocp-4-13-51-enhancements"]
+==== Enhancements
+
+* CENTOS 8 has recently ended its lifecycle. This release updates the CENTOS 8 references to CENTOS 9. (link:https://issues.redhat.com/browse/OCPBUGS-39160[*OCPBUGS-39160*])
```

Comment:
* CENTOS 8 has recently ended its lifecycle. This release updates the CENTOS 8 references to CENTOS 9. (link:https://issues.redhat.com/browse/OCPBUGS-39160[*OCPBUGS-39160*]) will be removed after the peer review as the note is not needed for 4.17. 4.16 already mentions this note.

Reply:
Wasn't sure what is meant in this comment about 4.17 since this is a 4.13.z update (maybe a copypasta leftover), but...

In general I'd be interested for more details on which "the CentOS 8 references" this is referring to. From the linked code PR in the Jira, it seems like this might make sense, if technically accurate:
```suggestion
* CentOS 8 has recently ended its lifecycle. This release updates the CentOS 8 references in the {product-title} installer to CENTOS 9. (link:https://issues.redhat.com/browse/OCPBUGS-39160[*OCPBUGS-39160*])
```
If this or a similar tweak is adopted, suggest applying to the other 4.y.z versions of this note in the related RNs as well.

Reply:
Sorry, confusion on my part. I had 4.17 living rent free in my mind (a term I've been meaning to use for some time). I updated the note as per your request and added the note back in. I've messaged the engineer just to confirm to re-addition of the note.